webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -47,13 +50,80 @@ const CompareList = ({ repo, reference, compareReference, after, onSelectRef, on
             <div className="tree-container">
                 {(results.length === 0) ? <Alert variant="info">No changes</Alert> : (
                     <Card>
-                        <Table borderless size="sm">
-                            <tbody>
-                            {results.map(entry => (
-                                <ChangeEntryRow key={entry.path} entry={entry} showActions={false}/>
-                            ))}
-                            </tbody>
-                        </Table>
+                        <Card.Header>
+                        <span className="float-left">
+                            {(delimiter !== "") && (
+                                <URINavigator
+                                    path={prefix}
+                                    reference={reference}
+                                    relativeTo={`${reference.id}...${compareReference.id}`}
```

Comment:
optional: trim if reference is a commit hash, to conform with a single commit view

Reply:
Will do